src/test/java/uk/nhs/prm/deduction/e2e/tests/EhrOutE2ETests.java

```diff
@@ -0,0 +1,247 @@
+import uk.nhs.prm.deduction.e2e.queue.BasicSqsClient;
+import uk.nhs.prm.deduction.e2e.queue.SqsMessage;
+import uk.nhs.prm.deduction.e2e.queue.ThinlyWrappedSqsClient;
+import uk.nhs.prm.deduction.e2e.queue.activemq.SimpleAmqpQueue;
+import uk.nhs.prm.deduction.e2e.transfer_tracker_db.TrackerDb;
+import uk.nhs.prm.deduction.e2e.transfer_tracker_db.TransferTrackerDbClient;
+import uk.nhs.prm.deduction.e2e.utility.LargeEhrTestFiles;
+import uk.nhs.prm.deduction.e2e.utility.Resources;
+import uk.nhs.prm.deduction.e2e.utility.TestUtils;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+
+import static org.awaitility.Awaitility.await;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+import static uk.nhs.prm.deduction.e2e.ehroutdb.tables.Acknowledgements.ACKNOWLEDGEMENTS;
+import static uk.nhs.prm.deduction.e2e.utility.TestUtils.*;
+
+@SpringBootTest(classes = {
+        EhrOutE2ETests.class,
+        TestConfiguration.class,
+        ThinlyWrappedSqsClient.class,
+        BasicSqsClient.class,
+        AssumeRoleCredentialsProviderFactory.class,
+        AutoRefreshingRoleAssumingSqsClient.class,
+        Resources.class,
+        TrackerDb.class,
+        Gp2gpMessengerQueue.class,
+        TransferTrackerDbClient.class,
+        RoleAssumingAwsConfigurationClient.class
+})
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class EhrOutE2ETests {
+    private static final Logger LOGGER = LogManager.getLogger(EhrOutE2ETests.class);
+    private final TrackerDb trackerDb;
+    private final Gp2gpMessengerQueue gp2gpMessengerQueue;
+    private final TestConfiguration config;
+
+    @Autowired
+    public EhrOutE2ETests(
+            TrackerDb trackerDb,
+            Gp2gpMessengerQueue gp2gpMessengerQueue,
+            TestConfiguration config
+    ) {
+        this.trackerDb = trackerDb;
+        this.gp2gpMessengerQueue = gp2gpMessengerQueue;
+        this.config = config;
+    }
+
+    @BeforeAll
+    void init() {
+        gp2gpMessengerQueue.deleteAllMessages();
+    }
+
+    @Test
+    void shouldPutASmallEHROntoRepoAndSendEHRToMHSOutboundWhenReceivingRequestFromGP() {
+        // Given
+        String inboundConversationId = UUID.randomUUID().toString();
+        String smallEhrMessageId = UUID.randomUUID().toString();
+        String outboundConversationId = UUID.randomUUID().toString();
+        String nhsNumberForTestPatient = "9727018440";
+        String previousGpForTestPatient = "M85019";
+        String asidCodeForTestPatient = "200000000149";
+        LOGGER.info(" ===============  outboundConversationId: {}", outboundConversationId);
+
+        SimpleAmqpQueue inboundQueueFromMhs = new SimpleAmqpQueue(config);
```

Comment:
Can we move this to an instance variable? :)

```suggestion
private static final SimpleAmqpQueue MHS_INBOUND_QUEUE = new SimpleAmqpQueue(config);
```